versions/ufo3/glyphs/glif.md

```diff
@@ -206,6 +206,14 @@ Authoring tools may convert instances of contours like this to anchor elements w
 {: #outline }
 ### outline: Outline description.
 
+Note that this element contains descriptions of graphical data. Therefore, 
+
+1. the order of contours and components may be relevant for other (hinting) data but has no inherent meaning.
```

Comment:
Kinda agree, but unsure if this is the right wording. Clearly contour order does matter for overlaps/winding direction.

Reply:
Thinking about this more, I think that I would drop this one too

Reply:
Agreed. The order of contours and components records useful information.